pinot-core/src/main/java/org/apache/pinot/core/segment/creator/impl/SegmentDictionaryCreator.java

```diff
@@ -211,6 +203,39 @@ public void build()
     }
   }
 
+  private void writeBytesValueDictionary(int numValues, byte[][] sortedByteArrays) throws IOException {
```

Comment:
java doc for this method. This is invoked for string and BYTES right?

Reply:
Done.